pooch/tests/test_hashes.py

```diff
@@ -45,43 +45,48 @@
 TINY_DATA_HASHES.update(TINY_DATA_HASHES_XXH)
 
 
-def test_make_registry():
+@pytest.fixture
+def pooch_tmp_path(tmp_path):
```

Comment:
```suggestion
def pooch_tmp_path(tmp_path):
    """
    Mirror the test data folder on a temporary directory. Needed to avoid
    permission errors when pooch is installed on a non-writable path.
    """
```